### ISSUE TITLE
Add app icon change from installed icon packs

### ISSUE THREAD

**Issue by RameshGokamalla** [amirzaidi/Launcher3#508]
Can you a feature to Select desired app icon from installed icon packs please? It's very nice feature that all other launchers have.

**Comment by Horak14** [comment]
Yeah. It's great to support icon Packs, but some of my favorite icons for certain apps are the alternative icons. Those don't show up by default. Launchers like Nova allow you to change each app's individual icon from the selected pack. Please add this, it's an awesome feature.
